src/ray/raylet/main.cc

```diff
@@ -106,6 +94,13 @@ int main(int argc, char *argv[]) {
   // IO Service for node manager.
   boost::asio::io_service main_service;
 
+  // Initialize stats.
+  const ray::stats::TagsType global_tags = {
+      {ray::stats::JobNameKey, "raylet"},
+      {ray::stats::VersionKey, "0.9.0.dev0"},
+      {ray::stats::NodeAddressKey, node_ip_address}};
+  ray::stats::Init(global_tags, metrics_agent_port, main_service);
```

Comment:
Currently, we share the io_service of each component to enable asynchronous metrics report. This will be same for all other cpp components. Please lmk if you are concerned about that it affects performance.